graylog2-web-interface/src/stores/streams/StreamsStore.ts

```diff
@@ -44,6 +45,28 @@ class StreamsStore {
         });
     return promise;
   }
+  searchPaginated(page, perPage, query) {
+    const url = PaginationHelper.urlGenerator(ApiRoutes.StreamsApiController.paginated().url, page, perPage, query);
+    const promise = fetch('GET', URLUtils.qualifyUrl(url))
+      .then(response => {
+        const pagination = {
+          count: response.pagination.count,
+          total: response.pagination.total,
+          page: response.pagination.page,
+          perPage: response.pagination.per_page,
+          query: response.pagination.query,
+        };
+        return {
+          streams: response.streams,
+          pagination: pagination,
```

Comment:
With [object shorthand](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015) you don't need to specify the variable if the key is the same.

```suggestion
          pagination,
```